doc/getting-started/apl-nuc.rst

```diff
@@ -313,8 +298,10 @@ Set up Reference UOS
 
    .. code-block:: none
 
-      # cd ~
-      # curl -O https://download.clearlinux.org/releases/26200/clear/clear-26200-kvm.img.xz
+      $ cd ~
+      $ mkdir uos
+      $ cd uos
+      $ curl -O https://download.clearlinux.org/releases/26200/clear/clear-26200-kvm.img.xz
```

Comment:
@ailun258 if we change the uos image download directory to ~/uos,  may be better to modify "-s 3,virtio-blk,/root/clear-26200-kvm.img" line in launch_uos.sh

Reply:
right, accepted